docs/web/experimental.css

```diff
@@ -67,80 +67,259 @@ color: black;
   src: url(https://fonts.gstatic.com/s/courgette/v8/wEO_EBrAnc9BLjLQAUk1VvoK.woff2) format('woff2');
   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
 }
-
-div.flex {
-/*  display:flex; justify-content:flex-start; align-items: flex-start;; flex-direction:row; flex-wrap: wrap;*/
-display: inline-grid;
-grid-template-columns: auto auto;
+/* 2022-01-29 ebb: CSS GRID DISPLAY: 
+ * OUTERMOST CONTAINER in the HTML document IS div.grid
+ * div.grid contains the div.sidebar and div.main
+ * We set a column display with relative size using fr units.
+ * fr units are meant to add up to 1 (as full width of the container)
+ * Here we set the first element (sidebar) with the smaller fr width
+ * and the second element (main) with the larger fr width. . */
+div.grid {
+display:grid;
+grid-auto-flow:column;
+grid-template-columns:.3fr .7fr;
+grid-column-gap: 2vw;
+/* grid-column-gap and grid-row-gap provide a little space in between
+ * cells of the grid. */
 }
 
-div.sidebar {position:sticky;
```

Comment:
@arrowarchive You definitely don't need sticky positioning with CSS grid set up properly on a container element. This was removed.

Reply:
Well it depends on how she wants the sidebar to look and behave.. she might want the sidebar to scroll down the side as the user scrolls so that it is always visible. In that case, I think she needs to keep the sticky on the sidebar. 

@arrowarchive you do still need to set the number of grid rows and columns you want your container elements to span across your page. See my link to the display:grid tutorial in my other comment!